python/common_validations.py

```diff
@@ -0,0 +1,28 @@
+import string_utils
+from parameters_validation import parameter_validation
+import re
+
+
+@parameter_validation
+def is_version(version: str):
+    if version is None or not version:
+        raise ValueError("version should be non-empty and should not be None")
+    if not re.match(r"\d+\.\d+\.\d+$", version):
```

Comment:
I think we can define a pattern like `CITUS_VERSION_PATTERN = "\d{1,2}\.\d{1,2}.\d{1,2}"` and use it in `is_version`, `is_tag` and in `MULT_EXT_x_SEARCH_PATTERN` vars

Reply:
Fixed by https://github.com/citusdata/tools/pull/91/commits/a2e8aef98e41c785588ddb62c2ed0f902bee180d